functions/exercise20/script.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+function getStringIdOfRandomChars(specifiedLength) {
+    const stringCharacters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+    let getStringIdOfRandomChars = "";
+    for (let i = 0; i < specifiedLength; i++) {
+        getStringIdOfRandomChars +=
```

Comment:
Ok, you've split into multiple lines, but I really ment that you should split it into multiple instructions, not only lines.